build_system/src/prepare.rs

```diff
@@ -116,17 +89,11 @@ fn prepare_libcore(
     patches.sort();
     for file_path in patches {
         println!("[GIT] apply `{}`", file_path.display());
-        let path = Path::new("../..").join(file_path);
+        let path = Path::new("../../..").join(file_path);
```

Comment:
Would it be clearer to use an absolute path?

Reply:
Hard to say...